frames/src/main/java/com/checkout/frames/component/expirydate/ExpiryDateComponent.kt

```diff
@@ -30,10 +47,43 @@ internal fun ExpiryDateComponent(
 @Preview(showBackground = true)
 @Composable
 private fun ExpiryDateComponentPreview() {
-    ExpiryDateComponent(
-        ExpiryDateComponentStyle(
-            InputComponentStyle(inputFieldStyle = InputFieldStyle(placeholderText = "Test placeholder"))
+    val style = InputComponentStyle(
+        titleStyle = TextLabelStyle(
+            "Expiry Date",
+            textStyle = TextStyle(size = 16, color = 0xFF141414, font = Font.SansSerif, maxLines = 1),
         ),
-        FramesInjector.create()
+        subtitleStyle = TextLabelStyle(
+            "Format is MM/YY",
+            textStyle = TextStyle(size = 12, color = 0xFF636363, font = Font.SansSerif, maxLines = 2),
+        ),
+        inputFieldStyle = InputFieldStyle(
+            textStyle = TextStyle(size = 16, color = 0xFF141414, font = Font.SansSerif, maxLines = 1),
+            placeholderText = "__ / __",
+            placeholderStyle = TextStyle(size = 16, color = 0x80636363, font = Font.SansSerif, maxLines = 1),
+            containerStyle = ContainerStyle(margin = Margin(top = 8, bottom = 8)),
+            indicatorStyle = InputFieldIndicatorStyle.Border(
+                unfocusedBorderColor = 0xFF8A8A8A,
+                focusedBorderColor = 0xFF0B5FF0,
+                errorBorderColor = 0xFFAD283E
+            )
+        )
     )
+    Column(modifier = Modifier.padding(20.dp)) {
+        Text(
+            text = "Payment details",
+            fontSize = 24.sp,
+            modifier = Modifier.padding(bottom = 20.dp),
```

Comment:
What are line 74 to 78 having hardcoded values ?

Reply:
Yes, it's just a preview component for us to test. It will be removed at the final stage.